### PR TITLE
app-emulation/virtualbox: v7.0.6 and v6.1.42

### PR DESCRIPTION
**Pull request by ceamac**
New upstream versions for both branches: 6.1 and 7.0.

<del>Work in progress.  All ebuilds build successfully, but I still have to check a few things:</del>
- <del>that they run (haven't started them yet)</del>
- <del>if any dependencies changed</del>
- <del>that they build with no flags and with all flags in an empty environment</del>
- <del>licenses for 6.1 branch</del>
- <del>if we have any recent updates to licenses for 7.0</del>

<del>Also I should add Python 3.11 to the 6.1 branch.</del>

I added a new `nls` USE flag to 7.0.6 because even if compiled headless, virtualbox would BDEPEND on `dev-qt/linguist-tools:5` for some translations, and that would pull in QT5, not ideal on a headless system.

### CODE REVIEW

File: app-emulation/virtualbox-guest-additions/virtualbox-guest-additions-6.1.42.ebuild [hunk 1]
```diff
@@ -0,0 +1,277 @@
...
+	)
+"
+DEPEND="
+	${RDEPEND}
+	>=dev-util/kbuild-0.1.9998.3127
+	>=dev-lang/yasm-0.6.2
+	sys-devel/bin86
+	sys-libs/pam
+	sys-power/iasl
+	x11-base/xorg-proto
```
Comment (juippis) [inline app-emulation/virtualbox-guest-additions/virtualbox-guest-additions-6.1.42.ebuild:48]: I see there's `X` use flag, **maybe** this is tied behind that one?

File: app-emulation/virtualbox-guest-additions/virtualbox-guest-additions-7.0.6.ebuild [hunk 2]
```diff
@@ -0,0 +1,287 @@
...
+"
+# some libs here are indirect dependencies, and also needed at compile time.
+# keeping them in DEPEND to avoid warnings from qa-vdb.
+DEPEND="
+	${RDEPEND}
+	x11-libs/libICE
+	x11-libs/libSM
+	x11-libs/libXau
+	x11-libs/libXdmcp
+	x11-base/xorg-proto
```
Comment (juippis) [inline app-emulation/virtualbox-guest-additions/virtualbox-guest-additions-7.0.6.ebuild:50]: Here too although it's `gui`.

File: app-emulation/virtualbox/virtualbox-7.0.6.ebuild [hunk 3]
```diff
@@ -0,0 +1,708 @@
...
+# but it needs more investigation.
+#
+# See bug #878299 to track this issue.
+DEPEND="
+	${COMMON_DEPEND}
+	>=dev-libs/libxslt-1.1.19
+	virtual/libcrypt:=
+	alsa? ( >=media-libs/alsa-lib-1.0.13 )
+	gui? (
+		x11-base/xorg-proto
```
Comment (juippis) [inline app-emulation/virtualbox/virtualbox-7.0.6.ebuild:98]: Looks good!

File: app-emulation/virtualbox-guest-additions/virtualbox-guest-additions-6.1.42.ebuild (hunk 1 above)
Comment (ceamac) [reply app-emulation/virtualbox-guest-additions/virtualbox-guest-additions-6.1.42.ebuild:48]: I think I should move it there.  I'll check this.  Thanks!

**Pull request closed without merging**